pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -249,62 +249,131 @@ protected static void validateAdminAccessForTenant(PulsarService pulsar, String
                                                        String originalPrincipal, String tenant,
                                                        AuthenticationDataSource authenticationData)
             throws Exception {
+        validateAdminAccessForTenantAsync(pulsar, clientAppId, originalPrincipal, tenant, authenticationData).get();
```

Comment:
Need unwrap CompletionException and ExecutionException.

Reply:
Done @Jason918